var/spack/repos/builtin/packages/py-snakemake-interface-report-plugins/package.py

```diff
@@ -15,6 +15,7 @@ class PySnakemakeInterfaceReportPlugins(PythonPackage):
 
     license("MIT")
 
+    version("1.1.0", sha256="b1ee444b2fca51225cf8a102f8e56633791d01433cd00cf07a1d9713a12313a5")
```

Comment:
Version `1.1.0` added a requirement

```python
snakemake = "^8.20.5"
```

Reply:
I was wondering how I missed that. This is a `dev` dependency, not a regular dependency. https://github.com/snakemake/snakemake-interface-report-plugins/blob/main/pyproject.toml#L14

Do you want to add these too?

Reply:
You are right.

It is a bit confusing. On one hand these plugins only do anything if `py-snakemake` is installed. On the other we didn't add Snakemake to dependencies so far.

Snakemake version is resolved indirectly through Snakemake interface packages. Let's make it consistent with how it was and ignore my comment about Snakemake constraint.